test/testnet/scripts/gen_constants.py

```diff
@@ -243,6 +245,258 @@ def gen_account_bundles_list(account_idx, seed):
     "9999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999"
     "9999999999999999999999999";
 
+//!
+//! Unused trytes to test PoW
+//!
+
+static const std::string UNUSED_TRYTES_1 =
```

Comment:
Are these trytes gonna work when regenerating entirely the DB?

Reply:
Yes, those trytes actually come from the mainnet.
They are just here in order to do pow testing, they do not have to be in the DB.
CI passed so I guess it's OK.